admin_interface/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,232 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-01-19 22:45-0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: admin.py:21
+msgid "Environment"
+msgstr "Ambiente"
+
+#: admin.py:30
+msgid "Language chooser"
+msgstr "Seletor de idioma"
+
+#: admin.py:37
+msgid "Logo"
+msgstr ""
```

Comment:
If the word is `Logo` in pt-br too, then the common practice would be to write it as the translated string.
It seems redundant but for other translators or tools that check translation files, it will be counted as translated instead of missing.

Reply:
Thanks for the sugestion. Done!